### ISSUE TITLE
Layout: ID is not recognized when used at update

### ISSUE THREAD

**Issue by jaejong** [primefaces-extensions/primefaces-extensions#371]
The test project depends on PF6.0 and PFE6.0.0 and works perfectly.
But When I upgrade PF be to above 6.0,  it is not updated.
[primefaces-test-master.zip](https://github.com/primefaces-extensions/primefaces-extensions/files/6365439/primefaces-test-master.zip)

**Comment by melloware** [comment]
OK let me investigate

**Comment by melloware** [comment]
Can you try my version here:
[pfe-371.zip](https://github.com/primefaces-extensions/primefaces-extensions/files/6367185/pfe-371.zip)

Its working fine for me in 10.0.0 ???

**Comment by jaejong** [comment]
Thank you for reply!
I tryed but it is not working for  me on following environment
os: wondow 8
java: jdk1.8.0_144
Can you let me know about your running environment?

**Comment by melloware** [comment]
```
Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
Maven home: C:\Tools\apache-maven-3.8.1
Java version: 1.8.0_211, vendor: Oracle Corporation, runtime: C:\Tools\jdk1.8.0_211\jre
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```

Tested on:
**Chrome 90**
**Firefox 88**

**Comment by jaejong** [comment]
I tried but not work on
Apache Maven 3.8.1
Java version: jdk1.8.0_291, vendor: Oracle Corporation
platform encoding: UTF8
OS name: "windows 8"

I see the defference between PF6.0 and PF10.0.0 at page source.
on PF6.0
layout id is same as xhtml
on PF1.0.0
layout id is assigned dynamically but reference static id=content

**on PF 6.0** 
<**div id="content"** style="width:100%; height:100%;"><div id="j_idt14" class="ui-layout-center ui-widget-content ui-corner-all pe-layout-pane-content" data-combinedposition="center"> 

<script id="content_s" type="text/javascript">$(function() {PrimeFaces.cw('ExtLayout', 'contentLayoutWidget',{id:'content',widgetVar:'contentLayoutWidget',forTarget:'#content',clientState:false,serverState:true,state:{},options:{"panes":{"minHeigh":"100%","resizable":true,"closable":true,"resizeWhileDragging":true,"resizeWithWindow":false,"slidable":false},"north":{"size":30,"resizable":false,"closable":false,"spacing_open":0},"west":{"resizeWhileDragging":false,"minWidth":150},"east":{"size":600},"center":{"resizeWhileDragging":true}}},true);});</script>


**on PF10.0.0**
<**div id="j_idt14"** class="ui-layout-center ui-layout-unit ui-panel ui-widget ui-widget-content ui-corner-all ui-layout-unit-content pe-layout-pane-content ui-panel-content" data-combinedposition="center">

<script id="content_s" type="text/javascript">$(function(){PrimeFaces.cw("ExtLayout","contentLayoutWidget",{id:"content",clientState:false,parent:"j_idt13",serverState:true,state:{},options:{"defaults":{"minHeigh":"100%","resizable":true,"closable":true,"resizeWhileDragging":true,"resizeWithWindow":false,"slidable":false},"north":{"size":30,"resizable":false,"closable":false,"spacing_open":0},"west":{"resizeWhileDragging":false,"minWidth":150},"east":{"size":600},"center":{"resizeWhileDragging":true}}});});</script></div><script id="fpc_s" type="text/javascript">$(function(){PrimeFaces.cw("ExtLayout","contentLayoutWidget",{id:"fpc",clientState:false,parent:"j_idt7",serverState:true,state:{},options:{"defaults":{"minHeigh":"100%","resizable":true,"closable":true,"resizeWhileDragging":true,"resizeWithWindow":false,"slidable":false},"north":{"size":30,"resizable":false,"closable":false,"spacing_open":0},"west":{"resizeWhileDragging":false,"minWidth":150},"east":{"size":600},"center":{"resizeWhileDragging":true}}});});</script>

**Comment by melloware** [comment]
Ahhh now I see your problem. I was looking for a CONSOLE errors.  You are just saying that the `update=` is not working.

You are right between 6.0 and 10.0.0 we heavily updated the Layout Component and its not assigning ID's if it a FullPage or considered Not Nested...
```java
public boolean isElementLayout() {
        return !isNested() && !isFullPage();
}
```

I fixed your reproducer to work in 10.0.0 see the attached version:
[pfe-371-2.zip](https://github.com/primefaces-extensions/primefaces-extensions/files/6376709/pfe-371-2.zip)

This is a duplicate of ticket: https://github.com/primefaces-extensions/primefaces-extensions/issues/125

**Comment by jaejong** [comment]
Thank you!!
Now it works.

**Comment by melloware** [comment]
Yep you just need to put outputPanel which is just a DIV you can update inside your LayoutPane's.
